deepaas/cmd/execute.py

```diff
@@ -0,0 +1,83 @@
+import argparse
+import shutil
+import ntpath
+import os
+import collections
```

Comment:
Imports must be in alphabetic order, and grouped as follows:

    <imports from stdlib>
    <blank-line>
    <imports from external modules>
    <blank-line>
    <imports from deepaas>